firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -50,12 +50,12 @@
 
   @NonNull
   public InstallationResponse createFirebaseInstallation(
-      long projectNumber,
+      @NonNull String projectID,
```

Comment:
minor note:
The order seems a bit random to me.
Why not start with the apiKey, which is a form of authentication, and then continue with all the request parameters (installationId, projectId, appId) ... or the other way around, first the parameters and then the apiKey...